test/hoss/utilities.py

```diff
@@ -57,8 +57,9 @@ def submit_and_download(harmony_client: Client, request: Request,
             replace(downloaded_filename, output_file_name)
             print(f'Saved output to: {output_file_name}')
 
-    except ProcessingFailedException:
+    except ProcessingFailedException as exception:
         print_error('Harmony request failed to complete successfully.')
+        raise exception
```

Comment:
I've changed this so that failed requests are more transparent, rather than swallowing the initial exception and then causing a different exception in something like `xarray` trying to open an empty file or something.

I've changed it in the `utilities.py` modules that have this function.